tests/scripts/run-xunit-perf.cmd

```diff
@@ -15,6 +15,7 @@ setlocal
   set TEST_CONFIG=Release
   set IS_SCENARIO_TEST=
   set USAGE_DISPLAYED=
+  set UPLOAD_TO_BENCHVIEW=0
```

Comment:
I don't quite understand.  You want me to change this because it has the same name as the command line flag that we are passing?